crates/dojo-core/src/database_test.cairo

```diff
@@ -8,9 +8,8 @@ use traits::{Into, TryInto};
 use starknet::syscalls::deploy_syscall;
 use starknet::class_hash::{Felt252TryIntoClassHash, ClassHash};
 use dojo::world::{IWorldDispatcher};
-use dojo::executor::executor;
-use dojo::database::{get, set, set_with_index, del, scan};
-use dojo::database::index::WhereCondition;
+use dojo::database::{get, set};
+use debug::PrintTrait;
```

Comment:
```suggestion
```

Reply:
Fixed in 28a083fc7e4f823b6fa0d7f1e9f5e2585ff4af95.